Editor/Modules/TextureModule.cs

```diff
@@ -0,0 +1,114 @@
+        MinMipMapLevel,
+        Resolution,
+        SizeOnDisk,
+        Path,
+        Num
+    }
+
+    class TextureModule : ProjectAuditorModule
+    {
+        public static string[] searchTheseFolders;                 // Manually Choose Folders to do the FindAssets search within.
+
+
+        static readonly IssueLayout k_IssueLayout = new IssueLayout
+        {
+            category = IssueCategory.Texture,
+            properties = new[]
+            {
+                // new PropertyDefinition { type = PropertyType.Description, name = "Texture Analysis", longName = "Results of Texture Analysis "},
+                new PropertyDefinition { type = PropertyTypeUtil.FromCustom(TextureProperties.Name), format = PropertyFormat.String, name = "Name", longName = "Show Texture Name" },
+                new PropertyDefinition { type = PropertyTypeUtil.FromCustom(TextureProperties.Format), format = PropertyFormat.String, name = "Format", longName = "Show Texture Format" },
+                new PropertyDefinition { type = PropertyTypeUtil.FromCustom(TextureProperties.Shape), format = PropertyFormat.String, name = "Shape", longName = "Show Texture Shape" },
+                new PropertyDefinition { type = PropertyTypeUtil.FromCustom(TextureProperties.ReadWrite), format = PropertyFormat.String, name = "Writeable", longName = "Show Texture ReadWrite" },
+                new PropertyDefinition { type = PropertyTypeUtil.FromCustom(TextureProperties.StreamingMipMaps), format = PropertyFormat.String, name = "StreamingMipMaps", longName = "Show Texture StreamingMipMaps" },
+                new PropertyDefinition { type = PropertyTypeUtil.FromCustom(TextureProperties.MinMipMapLevel), format = PropertyFormat.String, name = "MinMipMapLevel", longName = "Show Texture MinMipMapLevel" },
+                new PropertyDefinition { type = PropertyTypeUtil.FromCustom(TextureProperties.Resolution), format = PropertyFormat.String, name = "Resolution", longName = "Show Texture Resolution" },
+                new PropertyDefinition { type = PropertyTypeUtil.FromCustom(TextureProperties.SizeOnDisk), format = PropertyFormat.String, name = "SizeOnDisk", longName = "Show Texture SizeOnDisk" },
+                new PropertyDefinition { type = PropertyTypeUtil.FromCustom(TextureProperties.Path), format = PropertyFormat.String, name = "Location", longName = "Show Texture Location Path" }
+                //  new PropertyDefinition { type = PropertyType.Area, name = "Texture Recommendations", longName = "Recommendations for Optimizations for the Textures in the project." }
+                // above line makes stuff puke while the module is half-implemented
+            }
+        };
+
+        public override IEnumerable<IssueLayout> GetLayouts()
+        {
+            yield return k_IssueLayout;
+        }
+
+        public override void Audit(ProjectAuditorParams projectAuditorParams, IProgress progress = null)
+        {
+            // Implement your analysis here
```

Comment:
please remove unnecessary comments

Reply:
Solved in [Tweaks to Texture Module] commit.